book/embeds.md

```diff
@@ -572,7 +572,7 @@ modalities.
 
 ::: {.further}
 Videos are similar to images, but demand more bandwidth, time, and
-memory; they also have complications like [Digital Rights Management
+memory; they also have complications like [digital rights management
 (DRM)][drm]. The `<video>` tag addresses some of that, with built-in
```

Comment:
One thing I've noticed the copyeditor prefers "foo bar baz (FBB [N])", in other words they prefer just the acronym to be linked. But generally on the web I think it looks better to link the whole phrase, and in print it doesn't matter much. But we should standardize.

Reply:
agree; what do you prefer I do here?

Reply:
I think we should do whole phrase (keep what you have here) and add a bullet point to the Google Doc so when we do a final pass over things (after merging in all copyeditor changes) we make it look like this one.